### PR TITLE
enable knn

### PR DESCRIPTION
**Pull request by behrica**

### CODE REVIEW

File: test/tech/v3/libs/smile_ml_test.clj [hunk 1]
```diff
@@ -34,11 +34,11 @@
...
+ ;;   (doseq [regression-model smile-regression-models]
+ ;;     (verify/auto-gridsearch-regression {:model-type regression-model} 0.5)))
 
 
-;; (deftest smile-classification-autogridsearch-test
-;;   (doseq [classify-model smile-classification-models]
-;;     (verify/auto-gridsearch-classification {:model-type classify-model} 0.5)))
+ ;; (deftest smile-classification-autogridsearch-test
+ ;;   (doseq [classify-model smile-classification-models]
+ ;;     (verify/auto-gridsearch-classification {:model-type classify-model} 0.5)))
```
Comment (cnuernber) [inline test/tech/v3/libs/smile_ml_test.clj]: Are tests disabled?

Comment (behrica) [reply test/tech/v3/libs/smile_ml_test.clj]: These were always disabled. But they pass, you can enable them.

Comment (cnuernber) [reply test/tech/v3/libs/smile_ml_test.clj]: OK, I will enable them again after this lands; it was my mistake to have them disabled.  At the very least a nice sanity check.

**Comment by cnuernber** [comment]
All this needs is a quick update from master and it can land.

**Comment by behrica** [comment]
I rebased and enabled the grid tests.
As I did not specify mostly any hyperparameter ranges, the grid test will not do anything for the new models.

We need top get back to this at one point.

File: test/tech/v3/libs/smile_ml_test.clj (hunk 1 above)
Comment (behrica) [reply test/tech/v3/libs/smile_ml_test.clj]: I did so in my last commit.

**Pull request merged**